spec/app/domain/authentication/authn_k8s/validate_application_identity_spec.rb

```diff
@@ -197,7 +197,7 @@
         allow(host_annotation_2).to receive(:values)
                                       .and_return({ :name => "notimportant" })
 
-        expected_message = /Container authenticator was not found for requesting pod/
+        expected_message = /Container authenticator was not found in the pod/
```

Comment:
what is this syntax? "/" instead of a string?

Reply:
regex

Reply:
cool!